### PR TITLE
#184347009 Configure eslint and houndci

### PR DESCRIPTION
**Pull request by Fidela1**
### What does this PR do?

Configure ESLint and Integrate HoundCI

### Description of Task to be completed?

- Configure ESLint
- Integrate HoundCI with badge attached to readme

### How should this be manually tested?
1. Clone the [repo](https://github.com/atlp-rwanda/vikings-ec-bn)
2. Change into the directory vikings-ec-bn
3. Checkout branch `chore/configure-eslint-184347009`
4. Run the command `npm install`

 ### What are the relevant pivotal tracker stories?

[184347009](https://www.pivotaltracker.com/story/show/184347009)
[184347010](https://www.pivotaltracker.com/story/show/184347010)

### Screenshoots

N/A

### CODE REVIEW

**Review by irakozetony — APPROVED** [review]

**Review by Abdden — APPROVED** [review]

**Review by irakozetony — APPROVED** [review]

**Review by ndatumuremyi — APPROVED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: 'airbnb-base',
```
Comment (ncutixavier) [inline .eslintrc.js]: Why 'airbnb-base'?

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  overrides: [
+  ],
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+  },
+  rules: {
+  },
```
Comment (ncutixavier) [inline .eslintrc.js]: Please add some rules

**Review by ncutixavier — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 2 above)
Comment (ncutixavier) [reply .eslintrc.js]: Also fix the issue that is causing some files not to be reviewed by Hound CI

**Review by ncutixavier — CHANGES_REQUESTED** [review]
Why some files are not recognized by hound ci?

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**